packages/steps/src/BuildStepContext.ts

```diff
@@ -17,6 +18,7 @@ export class BuildStepContext {
     public readonly buildId: string,
     public readonly logger: bunyan,
     public readonly skipCleanup: boolean,
+    public readonly runtimePlatform: BuildPlatform,
```

Comment:
As for someone coming in with almost no knowledge about this part of the codebase it is a bit confusing to have entities like `Platform` and `BuildPlatform`, and have the variables of type `BuildPlatform` called `...platform` in one place, `...buildPlatform` in other and `...runtimePlatform` in another. I'm not saying it's wrong or even that there is a problem with that - in fact the naming makes sense, just that it can be a bit confusing for newcomers and maybe could be unified a little bit